pkg/dhcpcd/Makefile

```diff
@@ -1,15 +1,4 @@
-.PHONY: tag push
-default: push
+include ../package.mk
 
-ORG?=linuxkit
 IMAGE=dhcpcd
-DEPS=Dockerfile Makefile dhcpcd.conf usr/lib/dhcpcd/dhcpcd-hooks/10-mtu
-
-HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
-
-tag: $(DEPS)
-	docker build --squash --no-cache --network=none -t $(ORG)/$(IMAGE):$(HASH) .
-
-push: tag
-	DOCKER_CONTENT_TRUST=1 docker pull $(ORG)/$(IMAGE):$(HASH) || \
-	DOCKER_CONTENT_TRUST=1 docker push $(ORG)/$(IMAGE):$(HASH)
+DEPS=dhcpcd.conf usr/lib/dhcpcd/dhcpcd-hooks/10-mtu
```

Comment:
That'll pick up `10-mtu~` and `10-mtu.bak` and other editor detritus like that, won't it?

(that's generally why I prefer things like this to have a `.hook` suffix or something so you can match `*.hook`, but I suppose this is a dhcp implementation choice)